### DISCUSSION TITLE
Channel MTV 80 90 00 and MTV case ( all MTV )

### DISCUSSION THREAD

**Discussion by aliongminghao** [iptv-org/community#422] [Q&A] [answered]
Good morning. Is anyone know the url of MTV UK or us? From long time I've been searching MTV channel and I can't found it. 
I'm really appreciate for your help guys. Thank you very much.

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel
